.github/workflows/test.yml

```
@@ -2,9 +2,9 @@ name: Test and Coverage
 
 on:
   push:
-    branches: [main]
+    branches: [main,refactor]
```

YAMLlint (1.35.1)

[error] too few spaces after comma

(commas)